### DISCUSSION TITLE
Turning off blue led blink over sudo

### DISCUSSION THREAD

**Discussion by fpolli** [Joshua-Riek/ubuntu-rockchip#991] [Q&A] [answered]
I have an Orange Pi 5plus where I installed ubuntu 24.04 server from this distro. I do not have i/o devices attached to it and use ssh over my lan to control it from a different computer. Have had no problems until yesterday when I tried to stop the blue led from blinking after bootup (steady is fine). I followed the instructions in the Orange Pi wiki.

I ssh in:

ssh user@ipaddress

cd /sys/class/leds/blue_led
sudo echo none > trigger
-bash: trigger: Permission denied

When I checked the ls -la on that folder, trigger has -rwx r-- r-- root root permissions. I thought sudo ran a command as root. No idea why it's not working here. After web searches I tried ssh -t user@ipaddress and had no change.

Thanks

**Comment by Joshua-Riek** [comment, marked as answer]
Run the command as root with "sudo su" or look at the device tree overlays. There is an overlay to disable the leds.

**Comment by fpolli** [comment]
Thank you. I think the other suggestion I saw online looks a lot easier: duct tape
